llvm/include/llvm-c/Core.h

```diff
@@ -3434,6 +3441,13 @@ void LLVMMoveBasicBlockBefore(LLVMBasicBlockRef BB, LLVMBasicBlockRef MovePos);
  */
 void LLVMMoveBasicBlockAfter(LLVMBasicBlockRef BB, LLVMBasicBlockRef MovePos);
 
+/**
+ * Obtain the context to which this basic block is associated.
+ *
+ * @see llvm::Module::getContext()
```

Comment:
```suggestion
 * @see llvm::BasicBlock::getContext()
```

Reply:
Have removed the function and the associated comment.